### PR TITLE
feat(curriculum): make college algebra with python live

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by Sboonny** [comment]
there were missing dependencies issue, which is fixed in #49703, if your PR run into lint issue merge the latest branch 👍

**Review by raisedadead — APPROVED** [review]

File: config/superblock-order.ts [hunk 1]
```diff
@@ -322,11 +327,10 @@ export const superBlockOrder: SuperBlockOrder = {
         [SuperBlockStates.Legacy]: [SuperBlocks.RespWebDesign]
       },
       [TranslationStates.NotAudited]: {
-        [SuperBlockStates.Current]: [SuperBlocks.ProjectEuler],
+        [SuperBlockStates.Current]: [SuperBlocks.CollegeAlgebraPy, SuperBlocks.ProjectEuler],
```
Comment (naomi-lgbt) [inline config/superblock-order.ts:330]:
```suggestion
        [SuperBlockStates.Current]: [
          SuperBlocks.CollegeAlgebraPy,
          SuperBlocks.ProjectEuler
        ],
```

File: config/superblock-order.ts [hunk 2]
```diff
@@ -368,11 +373,10 @@ export const superBlockOrder: SuperBlockOrder = {
...
```
Comment (naomi-lgbt) [inline config/superblock-order.ts:376]:
```suggestion
        [SuperBlockStates.Current]: [
          SuperBlocks.CollegeAlgebraPy,
          SuperBlocks.ProjectEuler
        ],
```

File: config/superblock-order.ts [hunk 3]
```diff
@@ -463,11 +469,10 @@ export const superBlockOrder: SuperBlockOrder = {
         [SuperBlockStates.Legacy]: []
       },
       [TranslationStates.NotAudited]: {
-        [SuperBlockStates.Current]: [SuperBlocks.ProjectEuler],
+        [SuperBlockStates.Current]: [SuperBlocks.CollegeAlgebraPy, SuperBlocks.ProjectEuler],
```
Comment (naomi-lgbt) [inline config/superblock-order.ts:472]:
```suggestion
        [SuperBlockStates.Current]: [
          SuperBlocks.CollegeAlgebraPy,
          SuperBlocks.ProjectEuler
        ],
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Gotta please the linter gods

**Comment by naomi-lgbt** [comment]
This may be out of scope for the PR, but I think this is the first live superblock with the assignment-style challenges.

Is it intentional that there is no feedback on the question when the answer is correct but the assignment is not completed?

question

File: config/superblock-order.ts (hunk 1 above)
Comment (moT01) [reply config/superblock-order.ts]: should be resolved

**Review by naomi-lgbt — APPROVED** [review]
Would be happy to get this in

**Pull request merged**